cli/paas/client.go

```diff
@@ -75,13 +79,24 @@ func (c *CarrierClient) Apps() error {
 		return errors.Wrap(err, "failed to list apps")
 	}
 
-	msg := c.ui.Normal().WithTable("Name", "Status", "Route")
+	msg := c.ui.Normal().WithTable("Name", "Status", "Routes")
 
 	for _, app := range apps {
-		msg = msg.WithTableRow(app.Name)
+		status, err := c.kubeClient.StatefulSetStatus(
+			c.config.EiriniWorkloadsNamespace,
+			fmt.Sprintf("cloudfoundry.org/guid=%s", app.Name))
+		if err != nil {
+			return errors.Wrap(err, "failed to get app status")
```

Comment:
I wonder if we should not abort here, but simply show it in the table, in the status cell.
Note also, right now it does not tell the user for which app we got in trouble, and at this point no output was done yet, at all, leaving the user to guess.

Reply:
This should be an error when we can't talk to the cluster. I think that should be an immediate stop.

Reply:
I will include the app name in the error message though

Reply:
Thank you, that should be enough.